frontend/src/queries/nodes/DataTable/renderColumnMeta.tsx

```diff
@@ -21,7 +21,8 @@ export function renderColumnMeta(key: string, query: DataTableNode, context?: Qu
     let align: ColumnMeta['align']
 
     const queryContextColumnName = key.startsWith('context.columns.') ? trimQuotes(key.substring(16)) : undefined
-    const queryContextColumn = queryContextColumnName ? context?.columns?.[queryContextColumnName] : undefined
+    const queryContextColumn =
+        (queryContextColumnName ? context?.columns?.[queryContextColumnName] : undefined) ?? context?.columns?.[key]
```

Comment:
This was needed to get the `align` for the columns in the context to work. This fixed in the main error tracking scene as well.